app/features/feedback/lab/components/feedback-section.jsx

```diff
@@ -12,61 +17,147 @@ counterpart.registerTranslations('en', {
+    };
+
+  }
+
+  updateRules() {
+    this.props.saveRules(this.props.rules);
+  }
+
+  handleAllowedSuccessFeedbackMarkerColors(values) {
+    const newState = _.assign({}, this.state);
+    newState.allowedSuccessFeedbackMarkerColors = values;
+    this.props.rules.map(rule => {
+      rule.allowedSuccessFeedbackMarkerColors = newState.allowedSuccessFeedbackMarkerColors;
+    });
+    this.setState(newState, this.updateRules);
+  }
+
+  handleAllowedFailureFeedbackMarkerColors(values) {
+    const newState = _.assign({}, this.state);
+    newState.allowedFailureFeedbackMarkerColors = values;
+    this.props.rules.map(rule => {
+      rule.allowedFailureFeedbackMarkerColors = newState.allowedFailureFeedbackMarkerColors;
+    });
+    this.setState(newState, this.updateRules);
+  }
+
+  handleInputChange({target}) {
+    const newState = _.assign({}, this.state);
+    newState.form[target.name] = (target.type === 'checkbox')
+      ? target.checked
+      : target.value;
+    this.props.rules.map(rule => {
+      rule[target.name] = (target.type === 'checkbox')
+        ? target.checked
+        : target.value;
+    });
+    this.setState(newState, () => {
+      this.updateRules();
+    });
+  }
+
+  render() {
+    const {rules, deleteRule, editRule} = this.props;
+    const colorOptions = this.markerColorOptions;
 
+    return (<div className="feedback-section">
+      <Translate content="FeedbackSection.title" className="form-label" component="div"/>
+      <Translate content="FeedbackSection.help" className="form-help" component="small"/>
+
+      <div className="edit-feedback-global-options">
+
+        <CheckboxInput title={fieldText('pluralSuccessMessagesEnabled.title')} help={fieldText('pluralSuccessMessagesEnabled.help')} name="pluralSuccessMessagesEnabled" checked={this.state.form.pluralSuccessMessagesEnabled} onChange={this.handleInputChange.bind(this)}/>
+
+        <CheckboxInput title={fieldText('pluralFailureMessagesEnabled.title')} help={fieldText('pluralFailureMessagesEnabled.help')} name="pluralFailureMessagesEnabled" checked={this.state.form.pluralFailureMessagesEnabled} onChange={this.handleInputChange.bind(this)}/>
+
+        <CheckboxInput title={fieldText('colorizeUniqueMessagesEnabled.title')} help={fieldText('colorizeUniqueMessagesEnabled.help')} name="colorizeUniqueMessagesEnabled" checked={this.state.form.colorizeUniqueMessagesEnabled} onChange={this.handleInputChange.bind(this)}/> {this.state.form.colorizeUniqueMessagesEnabled && <SelectInput title={fieldText('allowedSuccessFeedbackMarkerColors.title')} help={fieldText('allowedSuccessFeedbackMarkerColors.help')} options={colorOptions} name="allowedSuccessFeedbackMarkerColors" multi={true} value={this.state.allowedSuccessFeedbackMarkerColors} placeholder={fieldText('allowedSuccessFeedbackMarkerColors.placeholder')} onChange={this.handleAllowedSuccessFeedbackMarkerColors.bind(this)}/>}
```

Comment:
Long lines like this should be flagged by eslint. Can you lint your changes, please?